packages/arb-token-bridge-ui/src/token-bridge-sdk/utils.ts

```diff
@@ -10,3 +11,46 @@ export const getChainIdFromProvider = async (provider: Provider) => {
   const network = await provider.getNetwork()
   return network.chainId
 }
+
+export const getProviderFromSigner = (signer: Signer) => {
```

Comment:
Ideally we shouldn't ever read `signer.provider` and should instead pass in the provider explicitly

Reply:
This was not being used anywhere, removed this function